raiden/messages.py

```diff
@@ -142,7 +142,7 @@ def message_from_sendevent(send_event: SendMessageEvent, our_address: Address) -
     elif type(send_event) == SendSecretReveal:
         message = RevealSecret.from_event(send_event)
     elif type(send_event) == SendBalanceProof:
-        message = Secret.from_event(send_event)
+        message = Unlock.from_event(send_event)
```

Comment:
It's really good the event had a good name and does not need to change or else we would be looking at DB upgrades.

Reply:
Now I see why DB upgrades were not necessary.

Reply:
Humm, this is a message, it does /not/ go into the database, it won't need a upgrade.

Reply:
My message was:
> Now I see why DB upgrades were **not** necessary.